_data/publications/povm-medrxiv.yml

```diff
@@ -3,3 +3,4 @@ link: https://www.medrxiv.org/content/10.1101/2020.06.29.20141283v1
 date: 2020-06-30
 citation: "Philippe Bourrianne, Stanley Chidzik, Daniel Cohen, Peter Elmer, Thomas Hallowell, Todd J. Kilbaugh, David Lange, Andrew M. Leifer, Daniel R. Marlow, Peter D. Meyers, Edna Normand, Janine Nunes, Myungchul Oh, Lyman Page, Talmo Periera, Jim Pivarski, Henry Schreiner, Howard A. Stone, David W. Tank, Stephan Thiberge and Christopher Tully, medRxiv https://doi.org/10.1101/2020.06.29.20141283"
 comment: medRxiv preprint
+focus-area: outreach
```

Comment:
Added this to put it somewhere, could be core or something else. @pelmer, @davidlange6 ?